x-team/xp-ui
# Create NotFoundScreen UI component

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3557

## Description

Create NotFoundScreen UI component

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3557-create-not-found-ui-layout-component` | [XP Registration](https://github.com/x-team/xp-registration/pull/220)

## Steps to Test or Reproduce

1. Check out the component wrapped with `ApplicantScreen` http://localhost:9001/?path=/story/screens-and-layouts-notfoundscreen--basic-usage

<img width="889" alt="Screen Shot 2020-06-24 at 10 17 41" src="https://user-images.githubusercontent.com/131859/85563821-1cf27e80-b604-11ea-8ce3-fbfecd72bfeb.png">

2. Check out the component wrapped with `ApplicantScreen` on **loading** state http://localhost:9001/?path=/story/screens-and-layouts-notfoundscreen-states--loading

<img width="888" alt="Screen Shot 2020-06-24 at 10 17 59" src="https://user-images.githubusercontent.com/131859/85563847-22e85f80-b604-11ea-95bb-e0ee4e9d2175.png">

3. Checkout the component without any wrapper http://localhost:9001/?path=/story/screens-and-layouts-notfoundscreen-debug--missing-props

<img width="893" alt="Screen Shot 2020-06-24 at 10 18 09" src="https://user-images.githubusercontent.com/131859/85563991-427f8800-b604-11ea-9f92-62244e6c54d1.png">

## Impacted Areas in Application

New component added `NotFoundScreen`
